server/drivers/NoritakeVFD.c

```diff
@@ -412,7 +412,7 @@ NoritakeVFD_chr (Driver *drvthis, int x, int y, char c)
  * \param string   String that gets written.
  */
 MODULE_EXPORT void
-NoritakeVFD_string (Driver *drvthis, int x, int y, const char string[])
+NoritakeVFD_string (Driver *drvthis, int x, int y, unsigned char string[])
 {
```

Comment:
It should be "const unsigned char" at least. However the API documentation quite clearly requires "const char" - maybe this is a bad idea. But then we should change this properly.